sycl/test/check_device_code/fp-accuracy.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl %s -ffp-accuracy=high -fno-math-errno \
```

Comment:
You also need `-fsycl-device-only` to avoid host compilation. I'd also recommend to structure RUN lines somewhat like this:
```
// RUN: %clangxx -fsycl -fsycl-device-only %s -fno-math-errno -ffp-accuracy-high \
// RUN:  -S -emit-llvm -o - | FileCheck %s
// RUN: %clangxx -fsycl -fsycl-device-only %s -fno-math-errno -ffp-accuracy-high \
// RUN: -ffp-accuracy=low:exp \
// RUN:  -S -emit-llvm -o - | FileCheck %s --check-prefix=CHECK-LOW-EXP
```

to ensure that common options appear the same and differences are easy to spot.

Maybe even use:
```
// DEFINE: %{common_opts} = -fsycl -fsycl-device-only -fno-math-errno -ffp-accuracy-high -S -emit-llvm -o -
// RUN: %clangxx %{common_opts} | FileCheck %s
// RUN: %clangxx %{common_opts} -ffp-accuracy=low:exp | FileCheck %s --check-prefix=CHECK-LOW-EXP
```

Reply:
@aelovikov-intel Thanks. Modified per your 2nd suggestion.